rules/robot_players.tex

```diff
@@ -80,6 +75,11 @@ \subsection{Team Markers}
   \label{fig:sponsor}
 \end{figure}
 
+\subsection{Goalkeeper}
+\label{sec:goalkeeper}
+
+The \emph{goalkeeper} may use any of the allowed jersey numbers and must use the same jersey number for the whole game. The \emph{goalkeeper} must wear a jersey with a primary color different from the primary colors used by the \emph{field players} of both teams.
```

Comment:
```suggestion
The \emph{goalkeeper} may use any of the allowed jersey numbers. The \emph{goalkeeper} must wear a jersey with a primary color different from the primary colors used by the \emph{field players} of both teams.
```
I would delete the restriction to one number for the goalkeeper. It is distinguishable through its color. And if a team wants to have more, it is okay.